src/renderers/common/CubeRenderTarget.js

```diff
@@ -72,6 +78,89 @@ class CubeRenderTarget extends WebGLCubeRenderTarget {
 
 	}
 
+	fromCubeTexture( renderer, cubeTexture, sigmaRadians = 0, poleAxis = new Vector3( 0, 1, 0 ) ) {
+
+		const currentGenerateMipmaps = cubeTexture.generateMipmaps;
+
+		cubeTexture.generateMipmaps = true;
+
+		this.texture.type = cubeTexture.type;
+		this.texture.colorSpace = cubeTexture.colorSpace;
+
+		this.texture.generateMipmaps = cubeTexture.generateMipmaps;
+		this.texture.minFilter = cubeTexture.minFilter;
+		this.texture.magFilter = cubeTexture.magFilter;
+
+		// The maximum length of the blur for loop. Smaller sigmas will use fewer
+		// samples and exit early, but not recompile the shader.
+		const MAX_SAMPLES = 20;
+
+		const blurMaterial = new NodeMaterial();
+		blurMaterial.side = BackSide;
+		blurMaterial.depthTest = false;
+		blurMaterial.depthWrite = false;
+		blurMaterial.blending = NoBlending;
+
+		const weights = uniformArray( new Array( MAX_SAMPLES ).fill( 0 ) );
+		const dTheta = uniform( 0 );
+		const n = float( MAX_SAMPLES );
+		const latitudinal = uniform( 0 ); // false, bool
+		const samples = uniform( 1 ); // int
+		const envMap = TSL_CubeTexture( null );
+
+		const cubeSampler = Fn( ( [ sampleDirection ] )=>{
+
+			return envMap.sample( sampleDirection );
+
+		} );
+		blurMaterial.fragmentNode = blur( { n, latitudinal: latitudinal.equal( 1 ), poleAxis: vec3( poleAxis ), outputDirection: positionWorldDirection, weights, samples, dTheta, sampler: cubeSampler } );
```

Comment:
@sunag thanks for getting the PMREM version of this blur working - now this one is giving me trouble. I'm getting a lot of exceptions like `getOutputNode( builder )` where `builder` is undefined. Also this line `device.queue.writeBuffer( bufferGPU, 0, buffer, 0 );` is giving me `TypeError: Overload resolution failed.` which sounds pretty bad. I'm going to try to make a simpler repro in a three.js example instead of MV.

Reply:
Okay, I think this is just async issues with WebGPURenderer in MV.